docs/main.md

```diff
@@ -0,0 +1,54 @@
+@page react-view-models react-view-models
+@parent can-ecosystem
+@description `react-view-models` connects observable view-models to [React](https://facebook.github.io/react/) [presentational components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8) to create auto-rendering [container components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8).
+@package ../package.json
+
+@body
+
+## Description
+
+`react-view-models` follows the pattern popularized by [react-redux](https://github.com/reactjs/react-redux) and provides users with a [react-view-models.reactViewModel reactViewModel] function for extending [React](https://facebook.github.io/react/) Presentational Components into Container Components, by providing a `ViewModel` constructor function, which is an extended [DefineMap](./can-define/map/map.html).
```

Comment:
I think breaking this sentence up into two sentences might make it easier to read.